spec/spec_helper.rb

```diff
@@ -67,6 +67,22 @@ def expect_shell_command(*command, exitstatus: 0, output: '')
   expect(Open3).to receive(:popen2e).with(*command).and_yield(mock_input, mock_output, mock_thread)
 end
 
+# If the `described_class` of a spec is a `Fastlane::Action` subclass, it runs
+# it with the given parameters.
+#
+def run_described_action(parameters, lane_name = 'test')
```

Comment:
```suggestion
def run_described_action(parameters, lane_name: 'test')
```
I'd suggest using a named parameters here especially when the previous arguments `parameters` is of variable length (in that case typically an implicit Hash, making it read like if it was a variadic number of named parameters)

Though even with a named parameter I'm curious how the call site would look like if you wanted to provided a different lane name. I think the only option at call site would be to make the Hash of parameters explicitly wrapped in `{ … }` in that case, to avoid having `… lane_nane: 'foo'` be considered part of the `parameters` Hash otherwise.

All that being said, I wonder what's the point of allowing to customize the lane name here, given it's only used as a dummy value to run it once parsed… so I think you can even totally remove it after all.

```suggestion
def run_described_action(parameters)
```